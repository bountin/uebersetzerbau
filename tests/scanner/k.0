a9b8c7d6
A8B8C7D6
