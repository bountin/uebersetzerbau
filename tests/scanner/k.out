ident a9b8c7d6
ident A8B8C7D6
